src/server/endpoints/signup-endpoints.js

```diff
@@ -89,6 +83,4 @@ function userSignup(req, res) {
   signupData.storeUser(req, res);
 }
 
-module.exports = {
-  userSignup: userSignup,
-};
+module.exports = {userSignup: userSignup,};
```

Comment:
find a better name, eg signupUser
userSignup is a noun, signupUser is an action